extensions/rich_text_components/components.py

```diff
@@ -40,7 +40,7 @@ class BaseRteComponent(object):
         'Filepath': objects.Filepath,
         'SanitizedUrl': objects.SanitizedUrl,
         'MathLatexString': objects.MathLatexString,
-        'ListOfTabs': objects.ListOfTabs,
+        'list': objects.ListOfTabs,
```

Comment:
This seems wrong. You don't want _every_ list in this and future RTE components to be treated as a ListOfTabs. I think you do want a custom object here.

Reply:
I am a bit confused here. If I define the schema for tab components in `objects.py`, instead of `rich_text_component_definitions.js`, tab is not rendered properly. I am not sure how should I change the schema so that it gets rendered properly. I tried looking in `protractor.js` but did not understand how should I make the changes. Can you guide me here, Thanks!

Reply:
Do whatever SanitizedUrl does. Similar principle.

Reply:
Done, PTAL!